simapp/app_config.go

```diff
@@ -24,14 +26,31 @@ import (
 	txconfigv1 "cosmossdk.io/api/cosmos/tx/config/v1"
 	upgrademodulev1 "cosmossdk.io/api/cosmos/upgrade/module/v1"
 	vestingmodulev1 "cosmossdk.io/api/cosmos/vesting/module/v1"
+
+	_ "cosmossdk.io/x/evidence"                       // import for side-effects
+	_ "cosmossdk.io/x/feegrant/module"                // import for side-effects
+	_ "cosmossdk.io/x/nft/module"                     // import for side-effects
+	_ "cosmossdk.io/x/upgrade"                        // import for side-effects
+	_ "github.com/cosmos/cosmos-sdk/x/auth/tx/config" // import for side-effects
+	_ "github.com/cosmos/cosmos-sdk/x/auth/vesting"   // import for side-effects
```

Comment:
this is so hood.

Reply:
Yeah, because we do not import the module directly with now there is now module app basic imports, we need this for app wiring.